plugin/documents.py

```diff
@@ -645,7 +651,6 @@ def _register_async(self) -> None:
         if not text_change_listener:
             debug("couldn't find a text change listener for", self)
             return
-        text_change_listener.view_listeners.add(self)
         self._text_change_listener_found = True
```

Comment:
This variable name, _text_change_listener_found, is now kind of confusing. Can we rename it to `_registered` or something?

Reply:
All it really needs to do is to prevent registering the view to the window over and over again.

Reply:
Changed to `_registered`.